src/main/java/fishstock/ui/MainWindow.java

```diff
@@ -0,0 +1,60 @@
+package fishstock.ui;
+
+import fishstock.Command;
+import fishstock.FishStock;
+import javafx.fxml.FXML;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    private static final String WELCOME_TEXT = "Hello, I'm FishStock.\nI might help if I feel like it.\n"
+            + "\nHere's a list of available commands:\n\n" + Command.toList();
```

Comment:
I like this line break because it does improve readability by breaking before the `+` operator.